packages/ui/src/2_molecules/WalletBalance/WalletBalance.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { SimpleTable } from '../SimpleTable';
+import styles from './WalletBalance.module.css';
+
+export const WalletBalance: FC<PropsWithChildren> = ({ children }) => (
```

Comment:
As I already mentioned, I believe this component should be in the frontend dapp itself since it just wraps `children` with `SimpleTable`. That way, you can do all of the logic in the new component.

We already discussed it together with @soulBit .